scripts/s3.py

```diff
@@ -214,7 +214,7 @@ def with_monitoring():
         tb = traceback.format_exc()
         now = datetime.now()
         tags = " ".join(telegram_users_to_alert)
-        message = f"`[{now}]`\n🔥 API (vaults) update for {Network(chain.id).name} failed!\n```\n{tb}\n```\n{tags}"
+        message = f"`[{now}]`\n🔥 API (vaults) update for {Network(chain.id).name} failed!\n```\n{tb}\n```\n{tags}"[:4000]
```

Comment:
https://core.telegram.org/bots/api#sendmessage
maybe increase to 4096 instead ?

Reply:
As discussed, tried with 4096, it failed, maybe due to emoji characters throwing things off. Will stick to 4000 to err or the safe side while still providing ample amounts of info